client/src/overview/components/Thumbnail.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-let Thumbnail = ({style, productStyles, setSelectedStyle}) => {
+let Thumbnail = ({productStyles, setSelectedStyle}) => {
 
   let changeStyle = (style) => {
     setSelectedStyle(prevState => style)
```

Comment:
@JeromeMTR  Not sure you need to reference prevState if you aren't updating the state based on the previous state.   I believe you can just set it to the current style. .

Reply:
Thats true, still good practice and performs the same result.

Reply:
@crypto-bender refactored!